utils/api/benefitHandler.ts

```diff
@@ -398,9 +406,17 @@ export class BenefitHandler {
         this.replaceTextVariables(result.eligibility.detail, result)
       )
 
+      // clawback is only valid for OAS
+      // This adds the oasClawback text as requested on ticket 90284.
```

Comment:
Leave out comments that refer to specific task as that might change, task can be closed/deleted, etc.